src/mlpack/core/util/param.hpp

```diff
@@ -64,7 +64,7 @@ using DatasetInfo = DatasetMapper<IncrementPolicy, std::string>;
  *     care of by CLI (however, you can explicitly specify newlines to denote
  *     new paragraphs).  You can also use printing macros like
  *     PRINT_PARAM_STRING(), PRINT_DATASET(), and others.
- * @param SEE_ALSOS A set of SEE_ALSO() macros that are used for generating
```

Comment:
Is this change acceptable? Doxygen was upset there was no literal reference to "SEE_ALSOS" in the signature.

Reply:
Ah, yeah, I think the `...` change is better.  I couldn't name the parameter in C macros though.